utils/common/src/main/java/org/apache/brooklyn/util/exceptions/Exceptions.java

```diff
@@ -461,4 +461,14 @@ public static String getMessageWithAppropriatePrefix(Throwable t) {
         return appendSeparator(getPrefixText(t), t.getMessage());
     }
 
+    /** Returns true if the root cause is a "boring" CNF, ie a straightforward declaration that the clazz is not found;
+     * this permits callers to include the cause only when it is interesting, ie caused by a dependent class not loadable.
+     */
+    public static boolean isRootBoringClassNotFound(Exception e, String clazz) {
+        Throwable root = Throwables.getRootCause(e);
+        if (!(root instanceof ClassNotFoundException)) return false;
```

Comment:
possibly, at least if it has no useful message.  i think maybe sometimes that one can have an interesting message?  wait until we see it occurring without clutter.